core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.spec.ts

```diff
@@ -237,6 +246,13 @@ describe('EditContentLayoutComponent with identifier', () => {
             contentType: 'test'
         });
     });
+
+    it('should have a [formData] reference on the <dot-edit-content-form>', async () => {
+        spectator.detectChanges();
+        const formElement = spectator.query('dot-edit-content-form');
```

Comment:
Use `testId`

Reply:
In this case is kinda ok... becaue you will never have two `dot-edit-content-form` in this component... technically.